### DISCUSSION TITLE
Unable to Access Chaos Mesh Dashboard

### DISCUSSION THREAD

**Discussion by Bhumika1997** [chaos-mesh/chaos-mesh#2023] [Q&A] [answered]
Hi I am Unable to launch dashboard for Chaos Mesh i ran the command " kubectl port-forward -n chaos-testing svc/chaos-dashboard 2333:2333 "  but it is stuck over here only not able to access through http://localhost:2333/ 
Can someone please help on this?

**Comment by AsterNighT** [comment]
Can you check chaos dashboard status? Run
```bash
kubectl describe pod chaos-dashboard -n chaos-testing
kubectl get svc -n chaos-testing
```

**Reply by Bhumika1997** [reply]
Status is Running
![image](https://user-images.githubusercontent.com/33874082/123217689-77107300-d4e8-11eb-8357-7185d651162f.png)
![image](https://user-images.githubusercontent.com/33874082/123217835-a0310380-d4e8-11eb-8a04-32ba4ca83ef9.png)
![image](https://user-images.githubusercontent.com/33874082/123217926-ba6ae180-d4e8-11eb-8f93-788f1cf7492e.png)

**Reply by AsterNighT** [reply]
And what is the actual behavior of the "stuck"? `kubectl port-forward` runs in the foreground and should freeze your shell. When you try to access `localhost:2333` you may see a prompt says "handling connection from 127.0.0.1:2333" or so.

**Reply by Bhumika1997** [reply]
When I try to access localhost:2333 it is showing site can't be reached
![image](https://user-images.githubusercontent.com/33874082/123235792-d32fc300-d4f9-11eb-9480-d54862f37a69.png)

**Reply by AsterNighT** [reply]
Can you try `netstat -pln` to check if port 2333 is actually listening?

**Reply by Bhumika1997** [reply]
![image](https://user-images.githubusercontent.com/33874082/123248086-29563380-d505-11eb-9ff2-0400beb1c07d.png)
![image](https://user-images.githubusercontent.com/33874082/123248197-43901180-d505-11eb-8298-6f52b58334f5.png)
![image](https://user-images.githubusercontent.com/33874082/123248285-5efb1c80-d505-11eb-993e-27d343fd4c50.png)

**Reply by AsterNighT** [reply]
I don't see any process listening on port 2333. Can you try rerun these steps? This time try run `kubectl port-forward` in the background.

```
kubectl port-forward -n chaos-testing svc/chaos-dashboard 2333:2333 &
netstat -pln | grep 2333
```

If it works fine, you should see the prompt telling you kubectl is forwarding the port and port 2333 is being listened on.

**Reply by Bhumika1997** [reply]
![image](https://user-images.githubusercontent.com/33874082/123253839-8b199c00-d50b-11eb-8e1e-590585ca8fc6.png)
![image](https://user-images.githubusercontent.com/33874082/123253942-ab495b00-d50b-11eb-8ad6-c9786354616e.png)

**Reply by AsterNighT** [reply]
That seems good, and can you still not access chaos-dashboard now?

**Reply by Bhumika1997** [reply]
No I can't access

**Reply by Bhumika1997** [reply]
Now it's working actually i was executing kubectl port-forward on the remote host called kmaster, and open browser on windows. 😰
I  should use kubectl port-forward --address=0.0.0.0 and visit <ip-of-kmaster>:2333 instead.
This worked for me now i am able to access dashboard

**Reply by Bhumika1997** [reply]
Thanku so much for your help @ @AsterNighT

**Comment by Bhumika1997** [comment, marked as answer]
Executing kubectl port-forward on the remote host called kmaster, and open browser on windows. 😰
Use kubectl port-forward --address=0.0.0.0 and visit :2333 instead.
This worked for me now i am able to access dashboard
